### PR TITLE
Downgrade Confluent to 1.9.2

### PR DESCRIPTION
**Pull request by joselsegura**
# Description

According to this [Confluent Kafka Python issue](https://github.com/confluentinc/confluent-kafka-python/issues/1519), using RHEL8 + Python 3.6 or 3.8 + latest confluent Kafka version can lead into a segmentation fault, observed by us in our stage environment.

Defaulting library to 1.9.2 solves the segmentation fault problem.

Fixes # (issue)

## Type of change

- Bug fix (non-breaking change which fixes an issue)

## Testing steps

Unit tests working, but need to reproduce in a real environment. I'm going to use an ephemeral environment

## Checklist
* [x] `make before_commit` passes
* [ ] updated documentation wherever necessary
* [x] added or modified tests if necessary
* [ ] updated schemas and validators in [insights-data-schemas](https://github.com/RedHatInsights/insights-data-schemas) in case of input/output change

### CODE REVIEW

File: ccx_messaging/consumers/kafka_consumer.py [hunk 1]
```diff
@@ -45,11 +43,19 @@ def __init__(
         )
 
         # Confluent initialization
-        config = {
-            "bootstrap.servers": kwargs.get("bootstrap_servers", ""),
-            "group.id": kwargs.get("group_id", None),
-            "retry.backoff.ms": retry_backoff_ms,
-        }
+        
+        config = kwargs.copy()
```
Comment (juandspy) [inline ccx_messaging/consumers/kafka_consumer.py:47]: why is copying from kwargs needed instead of just using the 3 args we need?

Comment (joselsegura) [reply ccx_messaging/consumers/kafka_consumer.py:47]: We don't know the amount of arguments that we will need in advance. Clowder configuration will inject, at least:

Comment (joselsegura) [reply ccx_messaging/consumers/kafka_consumer.py:47]: Given that, I'm trying to think in a long-term fix for the Clowder configuration mess that allow us to decouple the clowder configuration and the consumer and producers configuration

File: ccx_messaging/consumers/kafka_consumer.py [hunk 2]
```diff
@@ -45,11 +43,19 @@ def __init__(
...
+        config.update(
+            {
+                "bootstrap.servers": kwargs.get("bootstrap_servers", ""),
+                "group.id": kwargs.get("group_id", None),
+                "retry.backoff.ms": retry_backoff_ms,
+            }
+        )
+        if "bootstrap_servers" in config:
```
Comment (epapbak) [inline ccx_messaging/consumers/kafka_consumer.py:55]: not sure I understand these deletions. You're adding these values to the dict before, then deleting them if they exist

File: ccx_messaging/consumers/kafka_consumer.py (hunk 1 above)
Comment (juandspy) [reply ccx_messaging/consumers/kafka_consumer.py:47]: are all the clowder arguments compatible with confluent consumer/producer?

**Pull request closed without merging**